src/components/radio-group/radio-group.ts

```diff
@@ -39,6 +87,63 @@ export default class SlRadioGroup extends LitElement {
     });
   }
 
+  getAllRadios(): SlRadio[] {
```

Comment:
Should this be documented and exposed to the users?

Reply:
Nope. They can use `radioGroup.querySelector('sl-radio')`.